src/Databases/DatabaseReplicated.cpp

```diff
@@ -710,8 +710,9 @@ BlockIO DatabaseReplicated::tryEnqueueReplicatedDDL(const ASTPtr & query, Contex
 
 static UUID getTableUUIDIfReplicated(const String & metadata, ContextPtr context)
 {
-    bool looks_like_replicated = metadata.find("ReplicatedMergeTree") != std::string::npos;
-    if (!looks_like_replicated)
+    bool looks_like_replicated = metadata.find("Replicated") != std::string::npos;
```

Comment:
The substring search looks strange.

CREATE TABLE test (Replicated UInt8) ENGINE = Log

Reply:
There's a correct check a few lines below

Reply:
Maybe just delete these lines, so query will be parsed each time?